### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by micahlaster** [wrath16/MapPartyAssist#1]
could you eventually add tracking for
gil earned per map potentially per floor as well since the other stuff in this request is per floor
on shifting dungeons what each floor type was(including if it had a zap) and on doors what way the leader went
per floor how many thieves dropped as loot

**Comment by micahlaster** [comment]
by per floor i mean like when you hover over the checkmark displaying the results of that map per floor

**Comment by wrath16** [comment]
Yeah, I'll probably try to cover everything including loot and random events at some point but first release with stats will just cover the basics: what floors you got to, types of summons, and gil earned.
